guides/source/action_mailer_basics.md

```diff
@@ -857,17 +885,13 @@ files (environment.rb, production.rb, etc...)
 |`deliver_later_queue_name`|The name of the queue used with the default `delivery_job`. Defaults to the default Active Job queue.|
 |`default_options`|Allows you to set default values for the `mail` method options (`:from`, `:reply_to`, etc.).|
```

Comment:
We don't show configuration options in other guides I think.
My preference would be deleting this table and just link to the configuration guide, so we don't need to keep two guides in sync.